features/users/friendship.service.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+var debugEnabled = require("../../config").debug;
+var log = require("color-logs")(debugEnabled, debugEnabled, __filename);
+var Promise = require('promise');
+var Chance = new require('chance')();
+
+var User = require('./model');
+
+class FriendshipService {
+
+  async makeFriend(user_id, friend_id) {
+    try {
+      await User.updateOne(
+        { _id: user_id }, 
+        { $push: { "friends": friend_id } },
```

Comment:
- what happens if make the same friend many times? 
- what happens if I have 2,000 friends?

Reply:
refactor is not needed, just put your thoughts here if you have any ideas.